packages/image_picker/image_picker/lib/image_picker.dart

```diff
@@ -87,7 +87,8 @@ class ImagePicker {
   ///
   /// Use `preferredCameraDevice` to specify the camera to use when the `source` is [ImageSource.camera].
   /// The `preferredCameraDevice` is ignored when `source` is [ImageSource.gallery]. It is also ignored if the chosen camera is not supported on the device.
-  /// Defaults to [CameraDevice.rear].
+  /// Defaults to [CameraDevice.rear]. Since Android has no documented parameter for an intent to specify if the front or rear camera should be opened, this
```

Comment:
Maybe the `Since Android..` part should be a new line. Ditto below.

Reply:
Done, I updated the documentation to (also in the image_picker_platform_interface):

```
  /// Defaults to [CameraDevice.rear]. 
  /// 
  /// Note that Android has no documented parameter for an intent to specify if
  /// the front or rear camera should be opened, this function is not guaranteed
  /// to work on an Android device.
```